classes/class-sailthru-content.php

```diff
@@ -613,6 +620,52 @@ function sailthru_save_post( $post_id, $post, $update ) {
 		}
 	}
 
+	function sailthru_delete_post( int $post_id ) {
```

Comment:
This is basically the same method used to create posts with minimal changes to use it for deletion

Reply:
suggestion (non-blocking): Just thinking out loud. I wonder if we should pull out the reusable code. There are only a couple lines that are different. If we revisit these lines again, it might be worth the refactor.